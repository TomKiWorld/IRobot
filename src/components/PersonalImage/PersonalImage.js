import React from 'react';

const PersonalImage = ({ name }) => {
  return (
    <section>
      <h2 className='light-green'>Hello <span className='sega'>{name}</span></h2>
      <div className='tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5'>
          <img alt={`Sorry ${name}, Please try again`} src={`https://robohash.org/${name}?size=200x200`} />
      </div>
      <p className='light-green'>Meet your personal <span className='sega'>robot</span></p>
    </section>
  );
}

export default PersonalImage;
