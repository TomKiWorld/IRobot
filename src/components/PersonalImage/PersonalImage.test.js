import { shallow } from 'enzyme';
import React from 'react';
import PersonalImage from './PersonalImage';

describe('PersonalImage Component', () => {
  it('Expect to render PersonalImage Component', () => {
    expect(shallow(<PersonalImage />)).toMatchSnapshot();
  });
});
