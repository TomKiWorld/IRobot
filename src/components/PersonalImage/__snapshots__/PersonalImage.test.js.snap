// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalImage Component Expect to render PersonalImage Component 1`] = `ShallowWrapper {}`;
